app/uk/gov/hmrc/eoricommoncomponent/frontend/views/migration/enter_your_details.scala.html

```diff
@@ -54,8 +54,8 @@
         @CSRF.formField
 
         @if(UserLocation.isRow(selectedUserLocationWithIslands.getOrElse(""))){
-            <p class="govuk-hint" id="hint">@Html(messages("cds.matching.individual.hint"))</p>
-            <p class="govuk-body" id="hint-description">@Html(messages("cds.matching.individual.hint-description"))</p>
+            @paragraph(messages("cds.matching.individual.hint-description"))
+            @paragraph(messages("cds.matching.individual.hint"))
```

Comment:
Removed the govuk-hint class as per the ticket and swapped the order or the messages. Neither of these paragraphs should be hint text as they are not directly above the input fields